### PR TITLE
docs(index): add swot analysis

### PR DESCRIPTION
**Pull request by gabriosilva**
Based on this issue: #9

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -102,7 +102,25 @@ Planejador de trajetórias para voos em baixa altitude
...
+- Cliente consolidao
+### Fraquezas
+- Dependência de recursos financeiros públicos
+- Sistema Interno burocrático devido a área de atuação
+- Longo tempo de maturação de um profissional
+### Oportunidades
+- Alianças e cooperação com outras empresas do mesmo setor e países aliados ao brasil
+- Participação em programas governamentais
+- Expansão internacional
+- Investir na formação dos funcionários
```
Comment (gabriosilva) [inline docs/index.md:118]:
```suggestion
### Oportunidades
- Alianças e cooperação com outras empresas do mesmo setor e países aliados ao brasil
- Participação em programas governamentais
- Expansão internacional
- Investir na formação dos funcionários
- Alta Barreira para novos entrantes
```

File: docs/index.md [hunk 2]
```diff
@@ -102,7 +102,26 @@ Planejador de trajetórias para voos em baixa altitude
 
 ## Análise do cenário: Matriz SWOT
 
-*Matriz_SWOT*
+### Forças
+- Uso de indústrias locais para o fornecimento de matéria-prima
+- Soluções e tecnologias proprietárias e exlcusivas
+- Conhecimento consolidado do segmento
+- Cliente consolidao
```
Comment (gabriosilva) [inline docs/index.md:109]:
```suggestion
- Cliente consolidado
```

File: docs/index.md [hunk 3]
```diff
@@ -102,7 +102,26 @@ Planejador de trajetórias para voos em baixa altitude
...
+- Sistema Interno burocrático devido a área de atuação
+- Longo tempo de maturação de um profissional
+### Oportunidades
+- Alianças e cooperação com outras empresas do mesmo setor e países aliados ao brasil
+- Participação em programas governamentais
+- Expansão internacional
+- Investir na formação dos funcionários
+- Alta Barreira para novos entrantes
+### Ameaças
+- Mudanças no escopo poílitico nacional
```
Comment (gabriosilva) [inline docs/index.md:121]:
```suggestion
- Mudanças no escopo político nacional
```

File: docs/index.md (hunk 2 above)
Comment (IaagoTavares) [inline docs/index.md]:
```suggestion
- Maior cliente (Governo) possui uma estabilidade financeira, garantindo maior previsibilidade dos recursos 
```

**Review by IaagoTavares — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 4]
```diff
@@ -102,7 +102,26 @@ Planejador de trajetórias para voos em baixa altitude
 
 ## Análise do cenário: Matriz SWOT
 
-*Matriz_SWOT*
+### Forças
+- Uso de indústrias locais para o fornecimento de matéria-prima
```
Comment (gabriosilva) [inline docs/index.md:106]:
```suggestion
- Cadeia de fornecimento global para matéria-prima
```

**Comment by gabriosilva** [comment]
> Será que tem que usar o template daqui: https://drive.google.com/drive/folders/1lTqisJ_gxxtc5J5aXM8ula2kQ4MsII_M?

sim!! vou trocar aqui

**Review by gbcarneiro — APPROVED** [review]

**Review by dtonavitor — APPROVED** [review]

**Review by Lederback — APPROVED** [review]

**Pull request merged**